hera_qm/xrfi.py

```diff
@@ -161,33 +183,40 @@ def detrend_deriv(d, dt=True, df=True):
     return f
 
 
-def detrend_medminfilt(d, Kt=8, Kf=8):
+def detrend_medminfilt(d, flags=None, Kt=8, Kf=8):
     """Detrend array using medminfilt statistic. See medminfilt.
     Args:
         d (array): 2D data array of the shape (time, frequency) to detrend
+        flags (array, optional): 2D flag array to be interpretted as mask for d.
+            NOT USED in this function.
         Kt (int): size in time to apply medminfilter over
         Kf (int): size in frequency to apply medminfilter over
     Returns:
         float array: float array of outlier significance metric
     """
     if d.ndim != 2:
         raise ValueError('Input to detrend_medminfilt must be 2D array.')
-    d_sm = medminfilt(np.abs(d), 2 * Kt + 1, 2 * Kf + 1)
+    d_sm = medminfilt(np.abs(d), Kt=2 * Kt + 1, Kf=2 * Kf + 1)
```

Comment:
For readability, I generally prefer to put arguments that involve spaces in parens, e.g. `Kt=(2 * Kt + 1)`

Reply:
fixed